### DISCUSSION TITLE
Allow tapping keybinds to interact with pause menu buttons

### DISCUSSION THREAD

**Discussion by AlmightyDoor** [ppy/osu#15669] [Ideas] [answered]
Tapping keys on osu!standard work on continue/retry/quit buttons on stable but not on lazer. just a mild inconvenience

**Comment by peppy** [comment, marked as answer]
99% sure this is a duplicate, but you can also make it work by going into key bindings and binding your gameplay keys to be your "select" key also

**Reply by Joehuu** [reply]
Tracked in https://github.com/ppy/osu/issues/3551.
